core/modules/system/css/messages.theme.css

```
@@ -6,13 +6,14 @@
   position: relative;
   box-sizing: border-box;
   margin: 0.375rem 0;
-  padding: 0.9em 0.625em 1em 3.438em; /* LTR */
+  padding: 0.9em 36px 1em 3.438em; /* LTR */
   border-radius: 4px;
   min-height: 3.2em;
   overflow: hidden;
 }
 [dir="rtl"] .messages {
-  padding: .625em 3.438em .625em .625em;
+  padding-right:3.438em;
```

Missing space there:
```suggestion
  padding-right: 3.438em;
```